interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: when we have two+ vars that happen to have the same name, we use 'this' when we are referring to the global variable or the argument
 
-  Researched answer:
+  Researched answer: In most cases, the value of this is determined by how a function is called (runtime binding).
 
 
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: yarn allows us to run a react and test file when coding. A node_modules is added when you run yarn
 
-  Researched answer:
+  Researched answer: Yarn is a package manager for your code
 
 
 
 3. What are props in React?
 
-  Your answer:
+  Your answer: can be used as an object
 
-  Researched answer:
+  Researched answer: Props are basically kind of global variable or object
```

Comment:
Props are data passed between files in react. Props flow in one direction-- from parent to child component. Every child class component has props built into it where child functional components do not have props unless it is passed in.

Reply:
without the props, we wouldn't be able to access methods and values from the parent class